src/js/view/utils/submenu-factory.js

```diff
@@ -26,7 +26,9 @@ export const makeSubmenu = (settingsMenu, name, contentItems, icon, tooltipText)
         // Qualities submenu is the default submenu
         submenu = SettingsSubmenu(name, categoryButton, name === DEFAULT_SUBMENU);
         submenu.addContent(contentItems);
-        SimpleTooltip(categoryButtonElement, name, tooltipText);
+        if (!('ontouchstart' in window)) {
```

Comment:
@hghazzi is this to check that the device is mobile ? 'ontouchstart' is supported in chrome https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/ontouchstart

Reply:
`'ontouchstart' in window` is `false` in Chrome

Reply:
Will this always be the case? We may open ourselves up to a regression if they decide to change that in the future. I could see that happening since Chrome wants to unify their mobile/desktop platforms.